src/python/bot/fuzzers/afl/stats.py

```diff
@@ -233,10 +234,20 @@ def set_strategy_stats(self, fuzzing_strategies):
     |strategies|."""
 
     if fuzzing_strategies.use_corpus_subset:
-      self.stats[fuzzing_strategies.CORPUS_SUBSET_STRATEGY] = (
-          fuzzing_strategies.corpus_subset_size)
+      self.stats['strategy_{}'.format(
+          engine_common.CORPUS_SUBSET_STRATEGY.name)] = (
+              fuzzing_strategies.corpus_subset_size)
 
     if fuzzing_strategies.fast_cal == strategies.FastCal.MANUAL:
       self.stats[fuzzing_strategies.FAST_CAL_MANUAL_STRATEGY] = 1
     elif fuzzing_strategies.fast_cal == strategies.FastCal.RANDOM:
       self.stats[fuzzing_strategies.FAST_CAL_RANDOM_STRATEGY] = 1
+
+    if (fuzzing_strategies.generator_strategy == engine_common.Generator.RADAMSA
+       ):
+      self.stats['strategy_{}'.format(
```

Comment:
nit: the format seems unnecessary here (and everywhere else where we're doing the equivalent of concatenating a single value). just use '+'.